mm2src/coins/z_coin.rs

```diff
@@ -307,7 +315,8 @@ impl ZCoin {
             tx_builder.add_tx_out(output);
         }
 
-        let (tx, _) = tx_builder.build(consensus::BranchId::Sapling, &self.z_fields.z_tx_prover)?;
+        let (tx, _) =
+            tokio::task::block_in_place(|| tx_builder.build(consensus::BranchId::Sapling, &self.z_fields.z_tx_prover))?;
```

Comment:
For a reason that I don't remember, I couldn't use `async_blocking`. I will recheck this. Thanks for the tip!

Reply:
With `async_blocking`
```
error[E0759]: `self` has an anonymous lifetime `'_` but it needs to satisfy a `'static` lifetime requirement
    --> mm2src/coins/z_coin.rs:204:9
     |
204  |         &self,
     |         ^^^^^ this data with an anonymous lifetime `'_`...
...
319  |             async_blocking(|| tx_builder.build(consensus::BranchId::Sapling, &self.z_fields.z_tx_prover)).await?;
     |             -------------- ...is used and required to live as long as `'static` here
     |
note: `'static` lifetime requirement introduced by this bound
    --> /Users/artem/dev/atomicDEX-API/mm2src/common/common.rs:1106:31
     |
1106 |     F: FnOnce() -> R + Send + 'static,
     |                               ^^^^^^^

```
`block_in_place` has less strict bounds, which allows us to avoid moving or cloning here.